### DISCUSSION TITLE
Interfaçage Mon restau responsable

### DISCUSSION THREAD

**Discussion by JenniferStephan** [betagouv/ma-cantine#1391] [Autres sujets]
Communication csv et json des données utilisateurices monrestauresponsable
2 ème étape : échange technique

**Comment by JenniferStephan** [comment]
<img width="488" alt="Capture d’écran 2022-05-09 à 14 34 03" src="https://user-images.githubusercontent.com/36134318/168047695-9271274b-ca4f-448f-9ef6-1243514e6532.png">

**Comment by JeremyPinto** [comment]
Hello !
Je suis le développeur MonRestauResponsable.

Comme on se l'est dit, voici les données du questionnaire pour un utilisateur en json, et en csv.

J'envoie ici un screenshot du questionnaire pour que vous puissiez voir à quoi ça correspond dans l'interface.

Le but est d'identifier toutes les questions utiles pour ma-cantine, et connaitre notre "taux de remplissable" actuel si ma-cantine récupère ces données.

[mrr_export_questionnaire_78_1643016733.pdf](https://github.com/betagouv/ma-cantine/files/8701079/mrr_export_questionnaire_78_1643016733.pdf)
[mrr-survey.txt](https://github.com/betagouv/ma-cantine/files/8701105/mrr-survey.txt)
[2022-05-16_mrr_tous_questionnaires.xlsx](https://github.com/betagouv/ma-cantine/files/8701080/2022-05-16_mrr_tous_questionnaires.xlsx)

**Reply by JenniferStephan** [reply]
Hyper clair, merci bcp ! On regarde ça en équipe et on revient vers toi vite :)

**Comment by JeremyPinto** [comment]
Salut !
Avez-vous un retour à nous faire sur ce sujet ?
Merci,
